saleor/payment/gateways/stripe/stripe_api.py

```diff
@@ -24,10 +27,20 @@
 logger = logging.getLogger(__name__)
 
 
+@contextmanager
+def opentracing_trace(span_name):
```

Comment:
We could consider moving it to `saleor/core/tracing.py` to keep tracing stuff in one place.